internal/dashboard/model/inspection/job.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright (c) 2023 OceanBase
+ob-operator is licensed under Mulan PSL v2.
+You can use this software according to the terms and conditions of the Mulan PSL v2.
+You may obtain a copy of Mulan PSL v2 at:
+         http://license.coscl.org.cn/MulanPSL2
+THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+See the Mulan PSL v2 for more details.
+*/
+
+package inspection
+
+type JobStatus string
+
+const (
+	JobStatusSuccessful JobStatus = "successful"
+	JobStatusFailed     JobStatus = "failed"
```

Comment:
These status are used in report, thus the job must be in a finished status, but I think it's a good idea to add running and pending status as well, in case we may add apis to query job later on.